go/engine/login_provisioned_device.go

```diff
@@ -248,8 +260,35 @@ func (e *LoginProvisionedDevice) run(m libkb.MetaContext) (err error) {
 	return nil
 }
 
-func (e *LoginProvisionedDevice) connectivityWarning(m libkb.MetaContext) {
+func (e *LoginProvisionedDevice) suggestRecovery(mctx libkb.MetaContext) error {
+	// Whatever happens here won't result in a session, so we can mark this engine as a noop.
+	e.resetPending = true
+
+	enterReset, err := mctx.UIs().LoginUI.PromptResetAccount(mctx.Ctx(), keybase1.PromptResetAccountArg{
+		Text: "If you have forgotten your password and either lost all of your devices, or if you " +
```

Comment:
I think that this display text should be separated from the RPC argument and left to the caller to decide. It would be better if the arg contained a reason enum specifying the part of the flow the prompt originated from.